numpyro/examples/datasets.py

```diff
@@ -240,6 +237,21 @@ def _load_jsb_chorales():
     return processed_dataset
 
 
+def _load_higgs():
+    warnings.warn("Higgs is a 2.6 GB dataset")
+    _download(HIGGS)
+
+    file_path = os.path.join(DATA_DIR, 'HIGGS.csv.gz')
+    with io.TextIOWrapper(gzip.open(file_path, 'rb')) as f:
+        csv_reader = csv.reader(f, delimiter=',', quoting=csv.QUOTE_NONE)
+        obs = []
+        data = []
+        for row in csv_reader:
+            obs.append(row[0])
+            data.append(row[1:])
+    return np.stack(obs), np.stack(data)
```

Comment:
Do you intend to add an example? :D

Reply:
Yes, I will add a logistic regression for the HIGGS dataset with HMCECS. Maybe I'll add Consensus as well. Depends on other obligations.